bindings/netstandard/ElectionGuard/ElectionGuard.Decryption.Tests/ElectionGuard.Decryption.Tests.csproj

```diff
@@ -26,6 +25,7 @@
   <ItemGroup>
     <ProjectReference Include="..\ElectionGuard.Decryption\ElectionGuard.Decryption.csproj" />
     <ProjectReference Include="..\ElectionGuard.ElectionSetup\ElectionGuard.ElectionSetup.csproj" />
+    <ProjectReference Include="..\ElectionGuard.Encryption\ElectionGuard.Encryption.csproj" />

```

Comment:
in order to get this to work, i had to revert the use of the nuget package in the UI project and all the way down the stack. it was causing too many issues with resolving assemblies. it definitely makes sense to add it back when we can publish electionsetup and decryption as their own packages for reference by the application